upstream.Dockerfile

```diff
@@ -17,8 +17,8 @@ COPY go.sum go.sum
 COPY cmd cmd
 COPY util util
 COPY test test
-RUN make build
-RUN make build-util
+RUN CGO_ENABLED=0 make build
+RUN CGO_ENABLED=0 make build-util
```

Comment:
`make build-util` explicitly disables CGO (see the `bin/cpb` rule)

Reply:
Removed `CGO_ENABLED=0` from make build-util